resource_alks_iamrole_test.go

```diff
@@ -7,9 +7,9 @@ import (
 	"testing"
 
 	"github.com/Cox-Automotive/alks-go"
+	awspolicy "github.com/hashicorp/awspolicyequivalence"
```

Comment:
Just curious.  Is there a mechanical go reason you changed the order?

Reply:
`go fmt` actually did this